app/views/students/show.html.erb

```diff
@@ -4,7 +4,7 @@
       <div class="section">
         <% if current_user.is_admin_or_super_admin?  %>
           <div style="position: absolute; padding-left: 20px">
-            <%= link_to @user.former_student? ? former_students_path : students_path do %>
+            <%= link_to :back do %>
```

Comment:
If the user goes to the edit page, then goes back to the show page, wouldn't this button take them back to the edit page? The expected behavior here would be for the back button to always take them back to the students page.

Reply:
Apparently yes, just tested it. The problem is that you can get to the `students/show` view form the `candidates` view as well so we should redirect to the correct view. I'll leave it as it is and we can fix this bug after the demo since we'll want to show the candidates view.